### PR TITLE
[ci] Fix rubocop offense Lint/UnusedBlockArgument

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

**Comment by bgeuken** [comment]
@ChrisBr @adrianschroeter @hennevogel Please review

File: src/api/app/controllers/application_controller.rb [hunk 1]
```diff
@@ -448,7 +448,7 @@ def pass_to_backend( path = nil )
     render_error status: 408, errorcode: "timeout_error", message: exception.message
   end
 
-  rescue_from ActiveXML::ParseError do |exception|
+  rescue_from ActiveXML::ParseError do |_|
```
Comment (hennevogel) [inline src/api/app/controllers/application_controller.rb]: Do you have to pass a variable?

Comment (bgeuken) [inline src/api/app/controllers/application_controller.rb]: I wasn't sure. Let me check

Comment (bgeuken) [inline src/api/app/controllers/application_controller.rb]: Works even without the variable.

**Comment by bgeuken** [comment]
rebased again

**Comment by hennevogel** [comment]
Looks good to me :-) :fireworks:

**Pull request merged**